docs/webhooks.md

```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
 }
 </pre>
 
+
+### `user.availability.updated`
+
+Emitted when user's availability is updated from desk.
+
+<pre class="prettyprint lang-js">{
+    "eventId": "d36cd3c4-2d16-4a77-97c2-620bde859b29",
+    "eventType": "user.availability.updated",
+    "platform": "sd",
+    "projectId": 3030,
+    "clientId": 1,
+    "createdAt": "2017-04-22T11:01:00+02:00",
+    "sentAt": "2017-04-22T11:01:00+02:00",
+    "operatorId": 15253,
+    "availabilities": [{"communicationMode": "CHAT", "status": "AVAILABLE"}]
```

Comment:
When we have enumeration, we add a note with all enumeration possible below the payload.

For example, i could be:

```markdown
Please note :
| Attribute | Description |
| --- | --- |
| communicationMode | Could be CHAT, CALL, VIDEO or THIRD_PARTY |
| status | Could be AVAILABLE or UNAVAILABLE|
```